neutone_sdk/audio.py

```diff
@@ -1,15 +1,19 @@
 import base64
+from cffi import FFI
 from dataclasses import dataclass
 import logging
 import math
 import io
 import pkgutil
 import tempfile
 from typing import Optional, List, Union
+from typing_extensions import Self
```

Comment:
I didn't know about this, very useful, before I would be forced to write the type in quotes as a string

Reply:
Apparently they're adding it to the standard library with 3.11, but this typing_extensions package comes with pytorch